### ISSUE TITLE
Temurin 24 binary builds not accessible through API

### ISSUE THREAD

**Issue by jerboaa** [adoptium/api.adoptium.net#1077]
**Describe the bug**
Looking at https://github.com/adoptium/temurin24-binaries/releases/ I see [jdk-24+3-ea-beta](https://github.com/adoptium/temurin24-binaries/releases/tag/jdk-24%2B3-ea-beta) being present, however when trying to get it using:

```
wget https://api.adoptium.net/v3/binary/latest/24/ea/linux/x64/jdk/hotspot/normal/eclipse
```

I get a 404 error.

**To Reproduce**

```
$ wget https://api.adoptium.net/v3/binary/latest/24/ea/linux/x64/jdk/hotspot/normal/eclipse
--2024-06-24 16:55:27--  https://api.adoptium.net/v3/binary/latest/24/ea/linux/x64/jdk/hotspot/normal/eclipse
Resolving api.adoptium.net (api.adoptium.net)... 20.62.244.126
Connecting to api.adoptium.net (api.adoptium.net)|20.62.244.126|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-06-24 16:55:28 ERROR 404: Not Found.
```

**Expected behavior**
Get an EA build of JDK 24.

**Device (please complete the following information):**
 - OS: Linux
 - Browser: GNU Wget
 - Version: 1.21.3 built on linux-gnu

** Additional context **

[Last time we faced this issue with JDK 23](https://github.com/adoptium/api.adoptium.net/issues/840) it was a matter of redeploying the API endpoint with `-DLATEST_JAVA_VERSION=24`.

cc @johnoliver @zakkak @sxa

**Comment by jerboaa** [comment]
There seems to be a gap in our release process for every new JDK version when it comes to the API:

JDK 22: https://github.com/adoptium/api.adoptium.net/issues/720
JDK 23: https://github.com/adoptium/api.adoptium.net/issues/840
JDK 24: this issue :-/

**Comment by jerboaa** [comment]
Is there any ETA on this? @gdams Perhaps you could restart the API with `-DLATEST_JAVA_VERSION=24`?

**Comment by johnoliver** [comment]
@jerboaa resolved, will begin work on a fix for #1083 / #785
